doc/source/migrations/index.rst

```diff
@@ -22,7 +22,8 @@ Development version
 
 **Breaking changes:**
 
-- N/A
+- Upgrade default ``vale`` version from ``2.29.6`` to ``3.3.0`` in ``ansys/actions/doc-style`` action.
+  Vale config file ``.vale.ini`` and ``Vocab/ANSYS`` has to be changed as per `Vale section <https://dev.docs.pyansys.com/doc-style/formatting-tools.html#vale>`_ in PyAnsys developer’s guide
 
 **Migration steps:**
```

Comment:
@dipinknair - add a migration steps guide for moving to version 3 of vale.

Reply:
@RobPasMue  Added ! Please review.